embark.el

```diff
@@ -237,6 +237,10 @@ Embark comes with two such indicators:
   detailed information including key bindings and the first line
   of the docstring of the commands they run.
 
+- `embark-mixed-indicator', which combines the minimal and the
+  verbose indicator. The verbose popup is shown after
+  `embark-mixed-indicator-delay' seconds.
```

Comment:
Maybe the sentence starting with "The verbose popup..." needs to be on its own line?

Also, I think the compiler complains about a single space after a period.

Reply:
I'll take care of those after merging, @protesilaos.